New Design

Bashamega opened the pull request:
closes #1 
Major changes:

- New Design

1. Remove borders
2. Added shadow effects
3. Adde hover effects
4. Change the width of the buttons
5. Made the background of the _random_ button gradient
- Keyboard Shortcuts
- Changed `clalc.html` to `index.html`
- Added a file structure
![image](https://github.com/Code-With-Uttam/Advanced-Calc/assets/110662505/310d4216-2e63-4a4e-9e57-b8294d9032df)

Bashamega committed:
File structure

Bashamega committed:
CSS update

Bashamega committed:
Missing Closing tags

Bashamega committed:
Keyboard shortcuts

BwnnyRxbbit commented:
This looks awesome as always! Thank you!

BwnnyRxbbit commented:
Hi, I hate to have to ask you again but I added a dark mode function to this but I couldn't get it to right-align. Can you please do that?

Bashamega commented:
> Hi, I hate to have to ask you again but I added a dark mode function to this but I couldn't get it to right-align. Can you please do that?

I don't unerstand. The problem that i have noticed is that the text doesn't change. Thats all

BwnnyRxbbit commented:
> > Hi, I hate to have to ask you again but I added a dark mode function to this but I couldn't get it to right-align. Can you please do that?
> 
> I don't unerstand. The problem that i have noticed is that the text doesn't change. Thats all

Look [here](https://github.com/Code-With-Uttam/Advanced-Calc/commit/1dfef36f9aa29028eff3a056c784bd776b14b2dc), I have added a dark mode button but it is hidden under the Random number button.

Bashamega commented:
> > > Hi, I hate to have to ask you again but I added a dark mode function to this but I couldn't get it to right-align. Can you please do that?
> > 
> > 
> > I don't unerstand. The problem that i have noticed is that the text doesn't change. Thats all
> 
> Look [here](https://github.com/Code-With-Uttam/Advanced-Calc/commit/1dfef36f9aa29028eff3a056c784bd776b14b2dc), I have added a dark mode button but it is hidden under the Random number button.

You need to position it using css. I am going to do it

Bashamega commented:
check your discord @BwnnyRxbbit

BwnnyRxbbit commented:
> check your discord @BwnnyRxbbit

I'm at break in school and I can't be seen on disc. Please see [here](https://github.com/CelestialScript/CelestialScript/discussions/1)
